settings/README.md

```diff
@@ -211,6 +211,14 @@ use case can be interesting for example if some dialogs ask new values.
 return setting.set('limit', 20);
 ```
 
+Once executed, the settings panel will look like this and two messages will be
+in the console:
+
+![settings example](../_images/settings_example.png)
+
```

Comment:
Better to put the image in the settings folder rather that in ../_images/ folder to have a selfcontained example.

Reply:
I also like the idea the image is at the top of the README. In the meantime, @jtpio propose to put all images in the main README (#43) so this can stay as it is.